components/ProjectEditPage.tsx

```diff
@@ -476,13 +508,37 @@ export default function ProjectEditPage({ doc }: IProps) {
 								type="url"
 								className="border border-red-300 rounded-md px-1 ml-2 w-96"
 							/>
-						)
-				}
-				<TrashIcon className="w-6 h-6 ml-2 cursor-pointer" onClick={() => removeSubmissions(index)} />
+						</>
+					)
+						: ''}
+
+					<span>Message:</span>
```

Comment:
Messages are links to txt files.

Reply:
? I'm not sure I understand, submission.message has always been raw text string. I could expand submission.src as an optional text url field, but the front-end doesn't have rendering for it yet.

Reply:
Oh wait 🤦‍♂️ nvm. Ignore me being stupid